test/plugin/base_test.rb

```diff
@@ -674,8 +674,8 @@ def test_split_logs_by_tag
     end
     [
       # [] returns nil for any index.
-      [APPLICATION_DEFAULT_CONFIG, log_entry_count, dynamic_log_names, []],
-      [DISABLE_SPLIT_LOGS_BY_TAG_CONFIG, 1, [''], dynamic_log_names]
+      [SPLIT_LOGS_BY_TAG_ENABLE_CONFIG, log_entry_count, dynamic_log_names, []],
```

Comment:
Nit picky, but can we please flip these lines so their ordering matches the stub definition? If not, that's fine, I just found myself having a hard time reasoning about this change when the order was flipped.

Reply:
Sure thing. Flipped.